x-pack/plugins/infra/server/features.ts

```diff
@@ -20,11 +23,18 @@ export const METRICS_FEATURE = {
     all: {
       app: ['infra', 'kibana'],
       catalogue: ['infraops'],
-      api: ['infra', 'actions-read', 'actions-all', 'alerting-read', 'alerting-all'],
+      api: ['infra', 'actions-read', 'actions-all'],
       savedObject: {
         all: ['infrastructure-ui-source', 'alert', 'action', 'action_task_params'],
         read: ['index-pattern'],
       },
+      alerting: {
+        all: [
+          METRIC_THRESHOLD_ALERT_TYPE_ID,
+          METRIC_INVENTORY_THRESHOLD_ALERT_TYPE_ID,
+          LOG_DOCUMENT_COUNT_ALERT_TYPE_ID,
```

Comment:
In `infra` we have one codebase, but two applications (metrics and logs), the changes here for metrics look good, but I think `LOG_DOCUMENT_COUNT_ALERT_TYPE_ID` needs to be removed, and moved to the `LOGS_FEATURE` const instead. They get registered independently of one another.

Reply:
Great catch!
fixed